WMF Framework/UILabel+HasText.swift

```diff
@@ -0,0 +1,5 @@
+extension UILabel {
+    public var wmf_hasText: Bool {
+        return (text as NSString?)?.length ?? 0 > 0
```

Comment:
Yeah since that API is in flux, the labels are using NSStrings anyway, and this is faster than `characters.count`